contents/blog/the-truth-about-tiger-teams.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: What no one tells you about tiger teams
+date: 2025-11-29
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+    - natalia-amorim
+featuredImage: >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/hog_ql.png
+featuredImageType: full
+category: General
+tags:
+    - Culture
+    - Product
+seo:
+    metaTitle: 'What no one tells you about tiger teams'
+    metaDescription: "Tiger teams sound great in theory, but they rarely deliver. Here's why small teams beat tiger teams every time, and how to structure your org for speed."
+---
+
+There are a few core milestones in the lifecycle of a PostHog employee. 
+
+The first one is when your friends or family ask, "So… what does your company actually do?" and suddenly you're explaining the concept of B2B SaaS to your 87-year-old grandpa who just recently figured out how to operate Alexa.
+
+Once that's settled, one of two things happens: you've either bored them to death and they never ask about it again, or they double down on wanting to better understand how on earth you do what you do.
+
+"But how do you handle [that many products](/products)?!" 
+
+Congratulations, you've reached the next milestone: talking about [small teams](/small-teams). If they've ever worked in a large company, the response is predictable: "Oh okay, so like tiger teams."
+
+This is the moment where I take a deep breath, as the self-appointed president of the Tiger Team Hater Club™️.
+
+No.
+
+**Not** like a tiger team.
+
+Let me explain why.
+
+## What even is a tiger team?
+
+The term "tiger team" originated in the military (because of course it did) and was popularized by NASA, who famously assembled one during the Apollo 13 mission in 1970. Remember "_Houston, we have a problem_"? Well, they were the group tasked with dealing with said problem.
+
+It has since been usurped by corporations who wanted their problem-solving initiatives to sound less boring ("temporary cross-functional committee" doesn't have quite the same ring to it).
+
+A tiger team is typically a group of specialists and subject matter experts pulled into a smaller unit to tackle a specific problem, investigate a failure, or push through a critical project. The goal is to stay agile and deploy focused efforts when there's urgency – a deliberate contrast to the slow, bureaucratic pace of traditional org structures.
+
+The intent behind it is good. The execution rarely is.
+
+## Why tiger teams sound great but often aren't
+
+### 1. It's performative autonomy
```

Comment:
another thing i've seen (and heard reported)

folk who are doing the "boring normal" work while the tiger teams rush around resent the freedom and power the tiger team has

"why don't we get to fix the clink expander?! we were the people that figured out it's a problem"